invenio_oauthclient/contrib/openaire_aai.py

```diff
@@ -160,8 +161,13 @@ def get_rest_handlers(self):
 """OpenAIRE Remote REST Application."""
 
 # Sandbox
-_openaire_aai_sandbox_app = OpenAIREAuthSettingsHelper(
-    base_url="https://openaire-dev.aai-dev.grnet.gr",
+_openaire_aai_sandbox_app = KeycloakSettingsHelper(
```

Comment:
This will probably require a major release. Alternative is to create a new contrib.

Reply:
OpenAIRE has decomissioned `https://openaire-dev.aai-dev.grnet.gr` (the old URL for the sandbox). So nobody can use it and if in use it is failing. I think in this circumstances could even be a "patch" 😂, I would still do it as minor. WDYT?

Reply:
IMO still major, since we want to follow semver, I think it is regardless external condidtions :)